src/custom/abis/types/factories/GPv2Settlement__factory.ts

```diff
@@ -2,86 +2,92 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import { Contract, Signer, utils } from 'ethers'
-import { Provider } from '@ethersproject/providers'
-import type { GPv2Settlement, GPv2SettlementInterface } from '../GPv2Settlement'
+import { Contract, Signer, utils } from "ethers";
+import { Provider } from "@ethersproject/providers";
+import type {
+  GPv2Settlement,
+  GPv2SettlementInterface,
+} from "../GPv2Settlement";
```

Comment:
whats up with the semi colons?

Reply:
Auto generated code, will revert this file